src/main/java/spotify/models/albums/AlbumFull.java

```diff
@@ -10,6 +10,30 @@
 
 import java.util.List;
 
+/**
```

Comment:
What does a person reading this javadoc learn? There really is not any new info here.

You should write a short description what this class is used for and where it is used for.

Reply:
I will try it in the next PR request.